policies/elections.tex

```diff
@@ -34,58 +34,71 @@ \subsubsection{Power}
 \end{itemize}
+For further clarity, any number of weekends or other holidays may occur during any phase of the election, and such days shall not count towards the minimum or maximum length of each phase.
+So that any issues can be quickly addressed, when feasible the Elections Committee should aim to set the schedule such that no weekends or other holidays occurring during the voting period.
 
-The Elections Committee or Council shall set the entire schedule of an election before it begins, and the Committee shall advertise the dates.
+The Elections Committee or Council shall set the entire schedule of an election before it begins, and the Committee shall advertise the dates or arrange for MathSoc to advertise the dates.
 
 \subsection{Nominations}
 \subsubsection{Contents}
-All candidates for election must submit nominations to eligible for election.
-A nomination shall set out the following:
-\begin{itemize}
-	\item The name, userid, and contact information of the nominee;
-	\item A signed statement of the nominee agreeing to follow the elections procedure;
-	\item The seat and term for which the nominee is being nominated;
-	\item The names and userids of a number of Society members endorsing the nomination, each of whom is eligible to vote in the election to that seat; and
-	\item The signatures of each endorsee.
-\end{itemize}
-Five members shall be required to endorse a nomination for a Representative seat, and ten member shall be required for an executive seat (VPA, President). 
+All candidates for election must submit nominations to be eligible for election.
+A nomination shall set out at least one of the following:
+\begin{enumerate}
+	\item All of the following:
+		\begin{itemize}
+			\item The name, userid, and contact information of the nominee;
+			\item A signed statement of the nominee agreeing to follow the elections
+				procedure;
+			\item The seat and term for which the nominee is being nominated;
+			\item The names and userids of a number of Society members endorsing the
+				nomination, each of whom is eligible to vote in the election to that seat; and
+			\item The signatures of each endorsee.
+		\end{itemize}
+	\item All of the following:
+		\begin{itemize}
+			\item An online nomination completed on a voting platform endorsed by the Federation of Students and approved by the Elections Committee\footnote{for example, \votewebsite}
+		\end{itemize}
+\end{enumerate}
+Five members shall be required to endorse a nomination for a Representative seat.
+Ten members shall be required to endorse a nomination for an Executive seat.
 
 No member of the Elections Committee shall endorse a nomination, and no one shall endorse their own nomination.
+If a member of the Elections Committee endorses a nomination, the nomination shall be deemed invalid and the entire Elections Committee shall be informed of the error.
```

Comment:
If a member of the Elections Committee endorses a nomination, the nomination shall be deemed invalid and the entire Elections Committee, as well as the endorsed nominee, shall be informed of the error.